tests/test_datacube.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+import sys
+sys.path.append('../scripts/')
+from datacube import *
+
+def test_read_data(data_path):
```

Comment:
This seems like a reasonable thing to test. One thing that is fairly easy to do with pytest would be to check several different datapaths and make sure it behaves as expected on all of them, something like:

```
@pytest.mark.parametrize('data_path,expected_mv', [
    ('/path/to/good/data', False),
    ('/path/to/bad/data', True)])
def test_read_data(data_path, missing_values):
    missing_value = False
    
    ds = read_data(data_path)
    
    for var in ['dv', 'rlat', 'rlon', 'lat', 'lon']: 
        if var not in ds.variables:
            missing_value = True
    assert(missing_value == expected_mv)
```

Then if you ran `pytest` on the command line, it would run this test twice, once for each of the tuples in `parametrize`:
1. on `/path/to/good/data` and expect no values to be missing
2. on `/path/to/bad/data` and expect values to be missing